### DISCUSSION TITLE
Adobe Lightroom

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#73] [General]
# Adobe Lightroom

Check if Adobe Lightroom is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/6dc4947a-df41-4257-830e-86b0aca4dff1

**Comment by longc5** [comment]
I can't imagine this is also applicable to those ancient LR5, LR6 but need to ask. Thanks.
